build_sysext

```diff
@@ -14,42 +14,55 @@ SCRIPT_ROOT=$(dirname "$(readlink -f "$0")")
 assert_inside_chroot
 assert_root_user
 
+default_squashfs_template="$(readlink -f "${SCRIPT_ROOT}/../build/images")/<BOARD>/latest/${FLATCAR_PRODUCTION_IMAGE_SYSEXT_BASE}"
+
 # All these are used to set up the 'BUILD_DIR' variable
 DEFINE_string board "${DEFAULT_BOARD}" \
   "The board to build a sysext for."
-DEFINE_string build_dir '' \
-  "Directory used for building the sysext image, must exist, but should be empty."
 DEFINE_string metapkgs '' \
   "Comma-separated list of meta-packages to build from source and install into sysext image."
+DEFINE_string squashfs_base '' \
+  "The path to the squashfs base image. Defaults to the most current image built in '${default_squashfs_template}'."
 DEFINE_string manglefs_script '' \
   "A path to executable that will customize the rootfs of the sysext image."
-DEFINE_string squashfs_base '' \
-  "The path to the squashfs base image."
+DEFINE_boolean ignore_version_mismatch "${FLAGS_FALSE}" \
+  "Ignore version mismatch between SDK board packages and base squashfs. DANGEROUS."
```

Comment:
`<insert gif with rick dangerous>`

Reply:
> The situation with meta packages being passed in a different way than binary packages (a flag with comma-separated list as its value vs. a list of positional parameters) is a bit iffy, but I don't really have anything better to suggest right now.

I briefly considered replacing `--metapkgs` with a `--build-if-needed` option that builds the packages if necessary but that's just a different way of solving the issue. I think your approach is more flexible.